### ISSUE TITLE
delete unused tables

### ISSUE THREAD

**Issue by ahelsing** [GENI-NSF/geni-portal#1119]
The database has a bunch of OBE tables.

Looking at just the ones that have clearly no references:
```
account_slice
disabled_account
idp_account_request
institutions
institutions2
ma_member_id_translation: import_database script
tcm_slices
```

But there are others whose only reference is in portal db-util and are probably also OBE.

 _Imported from trac ticket [#1119](http://trac.gpolab.bbn.com/proto-ch/ticket/1119),  created by ahelsing on 07-29-2014 at 11:39, last modified: 03-31-2015 at 10:21_

**Comment by tcmitchell** [comment]
Closed by Pull Request #1652

**Comment by tcmitchell** [comment]
Closed by Pull Request #1652, but actually closing this time.
